pyprland/plugins/scratchpads/__init__.py

```diff
@@ -760,7 +760,7 @@ async def _handle_focus_tracking(self, scratch: Scratch, active_window: str, act
         if tracker and not tracker.prev_focused_window_wrkspc.startswith("special:"):
             same_workspace = tracker.prev_focused_window_wrkspc == active_workspace
             if scratch.have_address(active_window) and same_workspace and not scratch.have_address(tracker.prev_focused_window):
-                await self.hyprctl(f"focuswindow address:{tracker.prev_focused_window}")
```

Comment:
Is this change to scratchpads intended ?
(it'as about an expose bug... isn't it ?)

Reply:
Haha, it seems  I pushed to main instead of a  branch. You're right, this was totally an accident! I will fix it soon. :+1: